packages/crml-cennzx-spot/src/types.ts

```diff
@@ -12,92 +12,93 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import {AnyAddress, AnyAssetId, AnyNumber, IHash} from '@cennznet/types/types';
```

Comment:
What's difference between Hash and IHash?

Reply:
Interface renamed by polkadots to Hash

Reply:
Gotcha, thanks

Reply:
LGTM